cf/dynamo.yaml

```diff
@@ -33,10 +33,10 @@ Resources:
         - AttributeName: "experimentId"
           AttributeType: "S"
       KeySchema:
-        - AttributeName: "plotUuid"
-          KeyType: "RANGE"
         - AttributeName: "experimentId"
           KeyType: "HASH"
+        - AttributeName: "plotUuid"
+          KeyType: "RANGE"
```

Comment:
Hey Agi, does that mean that the order in which we specify the attributes here matters?

Reply:
yes, it turns out so, as Seb's error states : 

```
Invalid key order.  Hash Key must be specified first in key schema, Range Key must be specifed second
```

Reply:
I thought the order doesn't matter as it was a dictionary / map. Turns out it does,

Reply:
ah right, it does seem to matter.